### PR TITLE
disable package scan

### LINKED ISSUE #3128: warn instead of error if zero package catalogers are select - user might still run file metadata cataloger, for example
**What would you like to be added**:
I am trying to scan but don't use the packages tasks. is it possible to disable it and use only files tasks?
**Why is this needed**:
want to separate tasks
**Additional context**:

### PR DESCRIPTION
**Pull request by tomersein**
# Description
this PR enables the way to scan only using files cataloger without package cataloger
related to an older request I've submitted 3128 (link below)

- Fixes #3128

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] I have added unit tests that cover changed behavior
- [ ] I have tested my code in common scenarios and confirmed there are no regressions
- [ ] I have added comments to my code, particularly in hard-to-understand sections

### CODE REVIEW

**Comment by spiffcs** [comment]
Linked PR from @wagoodman. This is the direction we probably want to go where we elevate the file catalogers to be selected in the same way package catalogers are being used.

https://github.com/anchore/syft/pull/3505

TLDR we don't want to hide this and have it happen implicitly when `finalTasks` is  0.

@tomersein do you mind giving your feedback on 3505?

**Review by willmurphyscode — APPROVED** [review]
